docker/Dockerfile

```diff
@@ -110,7 +111,7 @@ RUN BOOST_VERSION=1.67.0 && \
         cxxflags=-w \
         install \
         && \
-    rm -rf boost*
+    rm -rf ${SCRATCH_DIR}
```

Comment:
```suggestion
    cd .. && rm -rf boost*
```

Reply:
That's one way. I opted for using `SCRATCH_DIR` to have the same approach as all other blocks in Dockerfile.